crates/findex/src/implementations/sqlite/findex.rs

```diff
@@ -121,7 +121,7 @@ impl FindexCallbacks<Error, UID_LENGTH> for SqliteFindex {
             .lock()
             .expect("Rusqlite connection lock poisoned");
         let tx = cnx.transaction()?;
-        for (uid, value) in items.iter() {
+        for (uid, value) in &*items {
```

Comment:
Is it the correction of a `clippy` lint ? I personally find it obfuscating, but I won't oppose `clippy`.